examples/multimodal_v1/components/worker.py

```
@@ -0,0 +1,290 @@
+        logger.info("VllmPDWorker has been initialized")
+
+    @endpoint()
+    async def generate(self, request: vLLMMultimodalRequest):
+        logger.debug(f"Received generate request in PDWorker: {{ id: {request.request_id} }}.")
+
+        if request.image_url is None:
+            # Process embeddings using the connector
+            embeddings, descriptor = self._embeddings_descriptor
+
+            if descriptor is None:
+                logger.error(f"in PD worker, descriptor is None")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)